grpcbp/client_middlewares.go

```diff
@@ -0,0 +1,65 @@
+package grpcbp
+
+import (
+	"context"
+	"strings"
+
+	"github.com/opentracing/opentracing-go"
+	"github.com/reddit/baseplate.go/ecinterface"
+	"github.com/reddit/baseplate.go/tracing"
+	"google.golang.org/grpc"
+)
+
```

Comment:
What about streaming interceptors?

Reply:
I think that should be added too but I would prefer to focus on getting something end-to-end working first unless we already have a great use case for streaming endpoints.